examples/docs_beta_snippets/docs_beta_snippets/dagster-plus/access/rbac/audit-logs.graphql

```diff
@@ -0,0 +1,17 @@
+query AuditLogQuery(
```

Comment:
In the interest of a "complete" example, I think it would be helpful to link or describe either how you would access the `/graphql` endpoint, or like how you would do it in python.

Reply:
Good idea. Added a brief sentence, can link out to the graphql page once that's ready.